pkg/kubelet/server/stats/summary.go

```diff
@@ -83,8 +85,11 @@ func (sp *summaryProviderImpl) Get() (*stats.Summary, error) {
 	if err != nil {
 		return nil, err
 	}
-
-	sb := &summaryBuilder{sp.fsResourceAnalyzer, node, nodeConfig, rootFsInfo, imageFsInfo, infos}
+	imageStats, err := sp.runtime.ImageStats()
+	if err != nil || imageStats == nil {
+		return nil, err
```

Comment:
If we are going down that path, I'd do it for all the stats. Or we can identify the ones that are required and ignore the optional ones. WDYT?
In any case, we can change that in a separate PR.